Adafruit_Video_Looper/video_looper.py

```diff
@@ -126,8 +128,50 @@ def _is_number(iself, s):
             return True
         except ValueError:
             return False
-    
+
     def _build_playlist(self):
+        """Try to build a playlist (object) from a playlist (file).
+        Falls back to an auto-generated playlist with all files.
+        """
+        if self._config.has_option('playlist', 'path'):
+            playlist_path = self._config.get('playlist', 'path')
+            if playlist_path != "":
+                if os.path.isabs(playlist_path):
+                    if not os.path.isfile(playlist_path):
+                        raise RuntimeError('Playlist path {0} does not exist.'.format(playlist_path))
+                else:
+                    paths = self._reader.search_paths()
+                    for path in paths:
+                        maybe_playlist_path = os.path.join(path, playlist_path)
+                        if os.path.isfile(maybe_playlist_path):
+                            playlist_path = maybe_playlist_path
+                            self._print('Playlist path resolved to {0}.'.format(playlist_path))
+                            break
```

Comment:
only m3u(8) files should be considered because there could be other files in there that are found before any m3u file in which case the script would terminate with Unrecognised playlist extension even if there is a m3u file present.